recipes/conversation_view/feed_item_row/feed_item_row.vue

```diff
@@ -0,0 +1,217 @@
+      >
+        <img
+          v-if="avatarImageUrl"
+          data-qa="feed-item-row-avatar"
+          alt=""
+          :src="avatarImageUrl"
+        >
+      </dt-avatar>
+      <!-- show time instead of avatar when headers not present -->
+      <div
+        v-if="!showHeader"
+        v-show="isActive"
+        class="d-fs-100 d-fw-normal d-ws-nowrap d-lh-100 d-fc-tertiary d-mb6"
+        data-qa="feed-item-row-time-left"
+      >
+        {{ shortTime }}
+      </div>
+    </template>
+
+    <article>
+      <!-- Feed Item -->
+      <div
+        v-if="showHeader"
+        class="d-d-flex d-ai-center"
+      >
+        <p class="d-fs-200 d-lh-300 d-fw-bold d-to-ellipsis d-of-hidden d-ws-nowrap">
+          {{ displayName }}
+        </p>
+        <time
+          class="d-fs-100 d-lh-300 d-fc-tertiary d-fw-normal d-ml4 d-fl-shrink0"
+        >
+          {{ time }}
+        </time>
+      </div>
+      <!-- Default content slot -->
+      <span class="content-text-wrapper-class">
+        <slot />
+      </span>
+    </article>
+
+    <template #bottom>
+      <div class="d-d-flex d-fw-wrap">
+        <slot name="reactions" />
+      </div>
+      <!-- Threads -->
+      <slot name="threading" />
+    </template>
+
+    <!-- Action menu -->
+    <template #right>
+      <div
+        v-show="isActive"
+        class="d-ps-absolute d-tn16 d-r12"
+      >
+        <dt-lazy-show
+          :appear="true"
+          transition="fade"
+          :show="isActive"
+        >
+          <slot name="menu" />
+        </dt-lazy-show>
+      </div>
+    </template>
+  </dt-list-item>
+</template>
+
+<script>
+import {} from './feed_item_row_constants.js';
+import { DtAvatar } from '@/components/avatar';
+import { DtLazyShow } from '@/components/lazy_show';
+import { DtListItem } from '@/components/list_item';
+
+export default {
+  name: 'DtRecipeFeedItemRow',
+
+  components: {
+    DtAvatar,
+    DtLazyShow,
+    DtListItem,
+  },
+
+  mixins: [],
+
+  inheritAttrs: false,
+
+  props: {
+    /**
+     * Show avatar, show header text or dont show left time and vice versa
+     */
+    showHeader: {
+      type: Boolean,
+      default: false,
+    },
+
+    /**
+     * Show the avatar of the user, if this is not passed in, the initials would display
+     */
+    avatarImageUrl: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * The display name of the sender
+     */
+    displayName: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * time string displayed as is.
+     * Shown on the header when showHeader is true
+     */
+    time: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * short time string without AM/PM displayed as is.
+     * Shown on the left of feed item when showHeader is false and isActive is true
+     */
+    shortTime: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * displays a darked background on the row.
+     */
+    isActive: {
```

Comment:
the JIRA says that isActive should continue to be true when hovered/focused, should this be done in this component with another data, or its responsibility of the component using this recipe to change isActive state by listening to the events?